src/main/java/org/spongepowered/api/util/command/args/GenericArguments.java

```diff
@@ -1126,8 +1119,8 @@ protected CatalogedTypeCommandElement(Text key, Game game, Class<T> catalogType)
         protected Iterable<String> getChoices(CommandSource source) {
             return this.game.getRegistry().getAllOf(this.catalogType).stream()
                 .map(input -> {
-                return input == null ? null : input.getId(); // TODO: ids or names?
-                })
+                        return input == null ? null : input.getId(); // TODO: ids or names?
```

Comment:
The indentation is awkward but it shuts up checkstyle. I'll put it back if you want.